ansible/configs/open-environment-azure-subscription/post_software.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Step 000 Post Software
+  hosts: localhost
+  connection: local
+  become: false
+  tags:
+    - step001
+    - post_software
+  tasks:
+    - debug:
+        msg: "Step 000 Post Software - Starting"
+
+    - name: Print Azure Subscription Based Sandbox Information
+      agnosticd_user_info:
+        msg:
+          - "We have temporarily allocated subscription {{ assignedsubscription['subscriptions'][0].subscription_id }} in the redhat0.onmicrosoft.com Azure organization to you."
+          - "You can log into https://portal.azure.com using your Red Hat SSO credentials."
+          - "NOTE: When this OPEN environment is deleted, the subscription will be reallocated to someone else and all of its resource groups will be deleted.  All data will be irrecovably removed."
+          - "Before retiring your OPEN environment, please try to clean up any App Registrations that you have created to reduce clutter in Azure."
+          - "Please regularly back up your data and script your changes in case you need to rebuild."
+          - "Please see this page for more information: https://www.opentlc.com/azure/openenv_sub_based_documentation.html"
+
+    - name: Print Azure Subscription Based OPEN Environment Details
+      agnosticd_user_info:
+        msg:
+          - ""
+          - "Your Azure environment details:"
+          - "Tenant ID: {{ azure_tenant }}"
+          - "Subscription ID: {{ assignedsubscription['subscriptions'][0].subscription_id }}"
+          - ""
+          - "Azure CLI quickstart:"
+          - "export GUID={{ guid }}"
+          - "export TENANT={{ azure_tenant }}"
+          - "export SUBSCRIPTION={{ assignedsubscription['subscriptions'][0].subscription_id }}"
+          - ""
+          - "curl -L https://aka.ms/InstallAzureCli | bash"
+          - ""
+          - "Login using web authentication:"
+          - "az login -u {{ email }} --tenant $TENANT"
+          - ""
+          - "See https://docs.microsoft.com/en-us/cli/azure/install-azure-cli for more info on installing the azure CLI"
+          - "See https://docs.microsoft.com/en-us/cli/azure/ for full documentation of the azure CLI"
```

Comment:
We should probably also have the azure tenant and subscription information in `agnosticd_user_info` `data`.

Reply:
done